src/gtkSourceView3.props

```diff
@@ -165,6 +166,13 @@ class SourceMark type "source_mark obj" set wrap wrapsig : GObject {
  "category"                 gchararray_opt  : Read / Write / Construct Only
 }
 
+class SourceMarkAttributes type "source_mark_attributes obj" set wrap wrapsig: GObject {
+"background" GdkRGBA  : Read / Write
```

Comment:
I've a question at this point: the corresponding C types is indeed `rgba` (apparently `color` is [deprecated](https://developer.gnome.org/gdk3/stable/gdk3-Colors.html)). However, in `gdk.mli`, `rgba` is an abstract type without any function for constructing values. Would it make sense to introduce a `Gdk.RGBA` module with more or less the same interface as `Gdk.Color` (At C level, [GdkRGBA](https://developer.gnome.org/gdk3/stable/gdk3-RGBA-Colors.html) features pretty much the same API as `GdkColor`)?

Reply:
I suppose it would make sense. I'm not sure how memory management is supposed to work, but it's small enough that it could not matter much.